### PR TITLE
Feature/list item refactor

### PR DESCRIPTION
**Pull request by seland**
#### Changes
 - Extracted ListItem component for common behavior.
 - Fixed NationsListItem, ChatListItem and SettingsListItem UI.
 - Removed unused assets.
 - Implemented ChatListItem according to designs.

#### Verification
 - CI should pass
 - Chat should look nicer

### CODE REVIEW

File: src/components/common/ChatListItem.js [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
   /**
-   * @desc Number of participants in the chat to be displayed.
+   * @desc Last message to show on preview.
    */
-  participants: string,
+  lastMessage?: string | null,
+  /**
+   * @desc Base64 avatar of partner.
+   */
+  avatar: any,
```
Comment (prameetc) [inline src/components/common/ChatListItem.js]: can we replace `any` with the type.

File: src/components/common/ListItem.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+   * @desc Text to show under main text.
+   */
+  subtitle?: string | null,
+  /**
+   * @desc Additional view to render on the right of the list item.
+   * @type string
+   */
+  AdditionalView: any,
+  /**
+   * @desc Flag to show/hide disclosure indicator.
```
Comment (prameetc) [inline src/components/common/ListItem.js]: same goes for `textStyle` and `AdditionalView`

File: src/components/common/ListItem.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+   */
+  disclosureIconVisible: boolean,
+  /**
+   * @desc Id that will be passed in onPress callback.
+   * @type string
+   */
+  id: any,
+  /**
+   * @desc Callback on press item.
```
Comment (prameetc) [inline src/components/common/ListItem.js]: I think `id` is a string

File: src/global/Styles.js [hunk 4]
```diff
@@ -604,6 +604,29 @@ const styles = {
...
+    justifyContent: 'space-between',
+    paddingBottom: 16,
+    paddingTop: 12,
+  },
+
+  chatTitleStyle: {
+    ...defaultTextStyles.body,
+    color: '#444444',
+    fontSize: 14,
+    fontWeight: '500',
```
Comment (prameetc) [inline src/global/Styles.js]: can the `color` attribute be taken from a global state by adding it to the global `Colors` variable? As it is repeated.

File: src/components/common/ListItem.js (hunk 3 above)
Comment (seland) [reply src/components/common/ListItem.js]: Generally, id can be really of any type, however it should be the same type across `id` and `onPress` first parameter, so I changed it to generic type.

File: src/components/common/ListItem.js (hunk 2 above)
Comment (seland) [reply src/components/common/ListItem.js]: Fixed.

File: src/components/common/ChatListItem.js (hunk 1 above)
Comment (seland) [reply src/components/common/ChatListItem.js]: Fixed.

**Pull request merged**